service/service.go

```diff
@@ -0,0 +1,79 @@
+package service
+
+import (
+	"context"
+	"github.com/ONSdigital/dp-frontend-renderer/assets"
+	"github.com/ONSdigital/dp-frontend-renderer/config"
+	renderHelpers "github.com/ONSdigital/dp-frontend-renderer/render"
+	"github.com/ONSdigital/dp-frontend-renderer/routes"
+	"github.com/ONSdigital/go-ns/handlers/requestID"
+	"github.com/ONSdigital/go-ns/handlers/timeout"
+	deprecatedLog "github.com/ONSdigital/go-ns/log"
+	"github.com/ONSdigital/go-ns/render"
+	"github.com/ONSdigital/log.go/log"
+	"github.com/gorilla/pat"
+	"github.com/justinas/alice"
+	unrolled "github.com/unrolled/render"
+	"html/template"
+	"net/http"
+	"time"
+)
+
+func Run(ctx context.Context, taxonomyRedirects map[string]string) error {
+	log.Namespace = "dp-frontend-renderer"
+
+	cfg, err := config.Get()
+	if err != nil {
+		log.Event(ctx, "unable to retrieve service configuration", log.Error(err))
+		return err
+	}
+
+	log.Event(ctx, "got service configuration", log.Data{"config": cfg}, log.INFO)
+
+	log.Event(ctx, "overriding default renderer with service assets", log.INFO)
+	render.Renderer = unrolled.New(unrolled.Options{
+		Asset:         assets.Asset,
+		AssetNames:    assets.AssetNames,
+		IsDevelopment: cfg.Debug,
+		Layout:        "main",
+		Funcs: []template.FuncMap{{
+			"humanSize":                renderHelpers.HumanSize,
+			"safeHTML":                 renderHelpers.SafeHTML,
+			"dateFormat":               renderHelpers.DateFormat,
+			"dateFormatYYYYMMDD":       renderHelpers.DateFormatYYYYMMDD,
+			"last":                     renderHelpers.Last,
+			"loop":                     renderHelpers.Loop,
+			"subtract":                 renderHelpers.Subtract,
+			"slug":                     renderHelpers.Slug,
+			"markdown":                 renderHelpers.Markdown,
+			"legacyDatasetDownloadURI": renderHelpers.LegacyDataSetDownloadURI,
+			"localise":                 renderHelpers.Localise,
+			"domainSetLang":            renderHelpers.DomainSetLang,
+			"taxonomyLandingPage": func(s string) string {
+				return taxonomyRedirects[s]
+			},
+		}},
+	})
+
+	router := pat.New()
+	alice := alice.New(
+		timeout.Handler(10*time.Second),
```

Comment:
Should this be config?

Reply:
Yes, but out of the scope of this ever-expanding PR which was just supposed to be a tiny PR into someone else's PR onto someone else and has turned into almost a complete refactor of the dp-frontend-renderer as it is lol.
However this particular matter was discussed in slack dev general (I was switching this to go modules at first but we axed that for now), we at some point need to remove this timeout.Hanlder, remove alice and pat and implement mux. For now, this is just a lift and shift and won't be changed in this PR